Packs/RTIR/Integrations/RTIR/RTIR.py

```diff
@@ -960,7 +960,7 @@ def main():
         elif demisto.command() == 'rtir-add-reply':
             add_reply()
 
-    except Exception, e:
+    except Exception as e:
```

Comment:
This is python 2 integration, not python 3

Reply:
This syntax works in BOTH.